### PR TITLE
DOC-175 Add RedisAI

### PR DESCRIPTION
**Pull request by bmansheim**
https://docs.redislabs.com/staging/DOC-175-add-redisai/modules/redisai/

### CODE REVIEW

**Comment by K-Jo** [comment]
@bmansheim we still need to have RedisAI installation instructions for enterprise as well

File: content/embeds/modules.html [hunk 1]
```diff
@@ -79,6 +79,25 @@ <h5>RedisGraph</h5>
             </section>
         </ul>
     </div>
+    <div class="home-options__option">
+        <div class="home-options__option__icon">
+            <img src="../theme-flex/img/ico-redisai.svg" alt="RedisGraph Icon" style="border:0px;width:80px">
```
Comment (K-Jo) [inline content/embeds/modules.html:84]: alt should be RedisAI

File: content/embeds/tryout-redisai.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
+## Getting Started
```
Comment (K-Jo) [inline content/embeds/tryout-redisai.md:1]: @bmansheim do we add this to others, the docker run?  I don't think we should do this here.  The idea is to get them to do it on redis enterprise

File: content/modules/redisai/release-notes/redisai-1.0-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+Title: RedisAI 1.0 Release Notes
+description:
+weight: 100
+alwaysopen: false
+categories: ["Modules"]
+---
+## RedisGears 1.0.0
```
Comment (K-Jo) [inline content/modules/redisai/release-notes/redisai-1.0-release-notes.md:8]: Should be redisai

File: content/embeds/tryout-redisai.md (hunk 2 above)
Comment (bmansheim) [reply content/embeds/tryout-redisai.md:1]: @K-Jo We didn't do it for other modules but currently we don't have installation instructions so maybe we change it when we add the installation instructions.

Comment (K-Jo) [reply content/embeds/tryout-redisai.md:1]: No we leave it out. thanks

**Comment by bmansheim** [comment]
@K-Jo We can put the RedisAI installation in the RedisAI section, no?

File: content/modules/redisai/redisai-quickstart.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+description:
+weight: 20
+alwaysopen: false
+categories: ["Modules"]
+---
+For this quick start tutorial, we recommend that you use:
+
+- Either:
+    - [A Redis Enterprise Software database with the RedisAI module enabled]({{< relref "/modules/create-database-rs.md" >}})
+    - [A Redis Cloud Pro database with the RedisAI module enabled](https://redislabs.com/redis-enterprise-cloud/)
```
Comment (K-Jo) [inline content/modules/redisai/redisai-quickstart.md:12]: we can't deliver this out of the box for now

File: content/modules/redisai/redisai-quickstart.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+Title: RedisAI Quick Start Tutorial
+description:
+weight: 20
+alwaysopen: false
+categories: ["Modules"]
+---
+For this quick start tutorial, we recommend that you use:
+
+- Either:
+    - [A Redis Enterprise Software database with the RedisAI module enabled]({{< relref "/modules/create-database-rs.md" >}})
```
Comment (K-Jo) [inline content/modules/redisai/redisai-quickstart.md:11]: this should later on also point to the installation instructions

File: content/modules/redisai/redisai-quickstart.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    - [A Redis Cloud Pro database with the RedisAI module enabled](https://redislabs.com/redis-enterprise-cloud/)
+    - A Docker container
```
Comment (K-Jo) [inline content/modules/redisai/redisai-quickstart.md:13]: @bmansheim I suggest to remove the docker one here as well

File: content/modules/redisai/_index.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+RedisAI provides the following data structures:
+
+- **Tensor**: Represents an n-dimensional array of values
+- **Model**: Represents a computation graph by one of the supported DL/ML framework backends
+- **Script**: Represents a [TorchScript](https://pytorch.org/docs/stable/jit.html) program
+
+## DL/ML Backends
+
+RedisAI supports the following DL/ML identifiers and respective backend libraries:
```
Comment (K-Jo) [inline content/modules/redisai/_index.md:83]: I would remove this and link to the release notes.  This contains as section which version are currently supported.  Or state that we are continuously adding backends, so latest info is in the release notes

File: content/modules/redisai/redisai-quickstart.md (hunk 5 above)
Comment (bmansheim) [reply content/modules/redisai/redisai-quickstart.md:11]: I added a note to https://redislabs.atlassian.net/browse/DOC-254 to add a link to here.

File: content/modules/redisai/redisai-quickstart.md (hunk 4 above)
Comment (K-Jo) [reply content/modules/redisai/redisai-quickstart.md:12]: @bmansheim I would rather have them reach out to us if they want this.  I meant that redisai is not zero-touch available from our website to create a database.  there is no set of instructions, it's more a manual operation our side

Comment (bmansheim) [reply content/modules/redisai/redisai-quickstart.md:12]: @K-Jo So I’ll take it out. Do we want to say something like “Contact Support if you want to try out RedisAI on Cloud”?

Comment (K-Jo) [reply content/modules/redisai/redisai-quickstart.md:12]: Yes please add this

**Pull request merged**